lib/models/questionnaire.dart

```diff
@@ -172,7 +162,7 @@ class Questionnaire {
 class QuestionnaireEntry<T extends Answer> {
   QuestionnaireEntry(this.question, [this.answer]);
 
-  final QuestionDefinition question;
+  QuestionDefinition question;
   final T? answer;
 
   bool get hasValidAnswer => answer?.isValid == true;
```

Comment:
This class should stay immutable/final. As discussed last time add a copyWith function to it.